tendenci/apps/memberships/views.py

```diff
@@ -2350,7 +2350,32 @@ def report_member_roster(request, template_name='reports/membership_roster.html'
 def report_member_quick_list(request, template_name='reports/membership_quick_list.html'):
     """ Table view of current members fname, lname and company only.
     """
-    members = MembershipDefault.objects.filter(status=1, status_detail="active").order_by('user__last_name')
+    order_by = request.GET.get('order_by', 'user__last_name')
+    
+    descending = order_by.startswith('-')
+    if descending:
+        order_by =  order_by[1:]
+    
+    members = MembershipDefault.objects.filter(status=1, status_detail='active')
+    
+    if order_by == 'member_number_int':
+        members = members.annotate(member_number_int=Cast('member_number', PositiveIntegerField()))
+
+    # If we want to be able order by an aggregated list of groups
+    # then we need to annotate the with that. Django provides no 
+    # generic means for creating such aggregates yet but the postgresql
+    # package provides one. So we can order by the group list only on
+    # postgresql systems for now. 
+    #
+    # Note the template can render a list without this annotation and
+    # shoudl fall on such a method.  
+    if connection.vendor == 'postgresql':
```

Comment:
Order by user groups list? That could cause performance issues and it doesn't seem to be useful.

Reply:
I find it very useful ;-).